### DISCUSSION TITLE
How to change mail via api & accessing user profile data

### DISCUSSION THREAD

**Discussion by PeterLinuxOS** [lvoegl/ScIdClient#2] [Ideas]
How I can access user profile data and also how I can change mail address, where I can find all endpoints for that?

**Comment by lvoegl** [comment]
Since this is not a public API and therefore also exists no documentation, you will have to take a look at the HTTP requests that the ingame client or website sends.
If you're lucky your endpoint may be in [this API documentation](https://github.com/ZeroBone/ScidApi/blob/master/APIDoc.MD) by ZeroBone.
